Seeing Error: Cannot find module '../package.json' when importing @traceloop/node-server-sdk

Hi I saw the following error when importing @traceloop/node-server-sdk in a nodejs playground in a backend project. (I was using llama index and following the official example to use OpenLLMetry)

```
Error: Cannot find module '../package.json'
Require stack:
- node_modules/@traceloop/instrumentation-chromadb/dist/src/instrumentation.js
- node_modules/@traceloop/instrumentation-chromadb/dist/src/index.js
- node_modules/@traceloop/node-server-sdk/dist/index.js
- playground/test-traceloop.cjs
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at Object.<anonymous> (node_modules/@traceloop/instrumentation-chromadb/dist/src/instrumentation.js:31:24)
    at Module._compile (node:internal/modules/cjs/loader:1233:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1287:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Module.require (node:internal/modules/cjs/loader:1115:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'node_modules/@traceloop/instrumentation-chromadb/dist/src/instrumentation.js',
    'node_modules/@traceloop/instrumentation-chromadb/dist/src/index.js',
    'node_modules/@traceloop/node-server-sdk/dist/index.js',
    'playground/test-traceloop.cjs'
  ]
}
```

Here is the code I use:
```typescript
import * as traceloop from "@traceloop/node-server-sdk";

traceloop.initialize({
  disableBatch: true,
  apiKey:
    "some key",
});

```

The tsconfig
```
{
  "compilerOptions": {
    "target": "ES2021",
    "module": "commonjs",
    "lib": ["es2021", "dom"],
    ...
   },
  "exclude": ["node_modules", "cdk.out"]
}

```

This prevents me from using OpenLLMetry in backend to track LLM use in Llama index. Any help is appreciated

--- a/packages/instrumentation-chromadb/package.json
+++ b/packages/instrumentation-chromadb/package.json
@@ -6,7 +6,7 @@
   "types": "dist/src/index.d.ts",
   "repository": "traceloop/openllmetry-js",
   "scripts": {
-    "build": "tsc --build tsconfig.json",
+    "build": "rollup -c",
     "lint": "eslint . --ext .ts",
     "lint:fix": "eslint . --ext .ts --fix",
     "test": "ts-mocha -p tsconfig.json 'tests/**/*.test.ts' --timeout 20000"

--- a/packages/instrumentation-chromadb/rollup.config.js
+++ b/packages/instrumentation-chromadb/rollup.config.js
@@ -1,0 +1,37 @@
+const dts = require("rollup-plugin-dts");
+const typescript = require("@rollup/plugin-typescript");
+const json = require("@rollup/plugin-json");
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const name = require("./package.json").main.replace(/\.js$/, "");
+
+const bundle = (config) => ({
+  ...config,
+  input: "src/index.ts",
+  external: (id) => !/^[./]/.test(id),
+});
+
+exports.default = [
+  bundle({
+    plugins: [typescript.default(), json.default()],
+    output: [
+      {
+        file: `${name}.js`,
+        format: "cjs",
+        sourcemap: true,
+      },
+      {
+        file: `${name}.mjs`,
+        format: "es",
+        sourcemap: true,
+      },
+    ],
+  }),
+  bundle({
+    plugins: [dts.default()],
+    output: {
+      file: `${name}.d.ts`,
+      format: "es",
+    },
+  }),
+];

